srcpkgs/mesa/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'mesa'
 pkgname=mesa
-version=20.2.3
+version=20.3.1
 revision=1
 wrksrc="mesa-${version}"
 build_style=meson
 configure_args="-Dglvnd=true -Dshared-glapi=true -Dgbm=true -Degl=true
  -Dosmesa=gallium -Dgles1=true -Dgles2=true -Dglx=dri -Ddri3=true
- -Dlmsensors=true -Dplatforms=x11,drm,$(vopt_if wayland wayland,)surfaceless
+ -Dlmsensors=true -Dplatforms=x11,$(vopt_if wayland wayland)
```

Comment:
Will this still build if one removes the wayland option?

Reply:
Yeah, it fails

> meson.build:21:0: ERROR: Options "" are not in allowed choices: "auto, x11, wayland, haiku, android, windows"

I'm okay with removing the build option :P

Reply:
Otherwise, `x11$(vopt_if wayland ,wayland)`

Reply:
Woops commented in the wrong spot. Yeah i thought about that too. Went ahead and did it. Should be good now